### DISCUSSION TITLE
Add a mention of split tunneling to `README.md`

### DISCUSSION THREAD

**Discussion by Korb** [DNSCrypt/dnscrypt-proxy#2739] [Ideas]
Based on issues #63 and #135, I assume that dnscrypt-proxy has splitting settings. I suggest mentioning it in `README.md` with a list of available features and limitations.

**Comment by jedisct1** [comment]
Can you clarify what "splitting settings" means?

**Reply by Korb** [reply]
> Can you clarify what "splitting settings" means?

Black and white lists of applications / processes and domains / addresses.
